ratpack-core/src/test/groovy/ratpack/http/client/HttpClientInterceptorSpec.groovy

```diff
@@ -62,6 +62,33 @@ class HttpClientInterceptorSpec extends BaseHttpClientSpec {
     result == 'foo'
   }
 
+  @Timeout(5)
+  def "can intercept errors"() {
```

Comment:
Make sure we add tests for verify that we can execute `Promise`s inside the `errorAction` just like we do with the response interceptor below (line 172)